doc/cephfs/experimental-features.rst

```diff
@@ -103,12 +102,20 @@ multiple active metadata servers is now permitted by default on new
 file systems.
 
 File Systems created with older versions of Ceph still require explicitly
-enabling multiple active metadata servers as follows:
-
-::
+enabling multiple active metadata servers as follows::
 
     ceph fs set <file system name> allow_multimds 1
 
 Note that the default size of the active mds cluster (``max_mds``) is
 still set to 1 initially.
 
+Which Ceph Version for Production?
```

Comment:
@batrick, Feels a little weird to the "Recommended Ceph Version" here. I'll still prefer this to be on the landing page.

Reply:
Hmm, perhaps I should change the page title to "Recommended Ceph Version"...

Reply:
@ceph/cephfs Any opinion?

Reply:
I think just remove this section entirely.

Reply:
Removed.